src/__tests__/__views__/Bookmark.test.js

```diff
@@ -0,0 +1,83 @@
+import "@babel/polyfill";
+import React from "react";
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import {
+  Bookmark,
+  mapStateToProps,
+  mapDispatchToProps
+} from "../../views/Bookmark";
+import { initialState } from "../../redux/reducers/bookmarkReducers";
+
+const setup = () => {
+  const props = {
+    bookmark: jest.fn(),
+    isBookmarked: false,
+    slug: "a-fake-slug",
+    bookmarkedArticles: {},
+    isSubmitting: false,
+    disabled: false
+  };
+  const enzymeWrapper = mount(<Bookmark {...props} />);
+
+  return {
+    props,
+    enzymeWrapper
+  };
+};
+
+describe("test bookmark component", () => {
+  const { props } = setup();
+  const bookmarkComponent = mount(<Bookmark {...props} />);
+  let instance;
+
+  beforeEach(() => {
+    instance = bookmarkComponent.instance();
+    jest.spyOn(instance.props, "bookmark");
+    instance.forceUpdate();
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("snapshot testing for the component", () => {
```

Comment:
Why are using `it()` yet other files are using `test()`?

Reply:
FIxed